contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -154,26 +124,27 @@ contract LPOptimiserStrategy is DefaultAccessControl {
 
     /// @notice Set new optimimal tick range based on current tick
     /// @param currentFixedRateWad currentFixedRate which is passed in from a 7-day rolling avg. historical fixed rate.
-    function rebalance(uint256 currentFixedRateWad) public returns (int24 newTickLowerMul, int24 newTickUpperMul) {
+    /// @return newTickLowerMul The new lower tick for the rebalanced position
+    /// @return newTickUpperMul The new upper tick for the rebalanced position
+    function rebalanceTicks(uint256 currentFixedRateWad) public returns (int24 newTickLowerMul, int24 newTickUpperMul) {
         _requireAtLeastOperator();
 
         // 0. Get tickspacing from vamm
-        // _tickSpacing = _vamm.tickSpacing(_vamm);
+        int24 tickSpacing = _vamm.tickSpacing();
 
         // 1. Get the new tick lower
-        // write UTs to check for underflow
         int256 deltaWad = int256(currentFixedRateWad) - int256(_sigmaWad);
-        int256 newFixedLowerWad = 0;
-        if (deltaWad > 0) {
-            // delta is greater than 0 => choose delta
-            if (deltaWad < _max_possible_lower_bound_wad) {
+        int256 newFixedLowerWad;
+        if (deltaWad > 1e15) {
+            // delta is greater than 1e15 (0.001) => choose delta
+            if (deltaWad < _maxPossibleLowerBoundWad) {
                 newFixedLowerWad = deltaWad;
             } else {
-                newFixedLowerWad = _max_possible_lower_bound_wad;
+                newFixedLowerWad = _maxPossibleLowerBoundWad;
             }
         } else {
-            // delta is less than or equal to 0 => choose 0
-            newFixedLowerWad = 0;
+            // delta is less than or equal to 1e15 (0.001) => choose 1e15 (0.001)
+            newFixedLowerWad = 1e15;
```

Comment:
below on lines 153 and 158 where we do fixed rate to tick conversions isn't it cleaner to use the reverse logic of this https://github.com/Voltz-Protocol/voltz-core/blob/72a2b1f725ed53340046ab02de3eff4ba9cdd4bf/contracts/MarginEngine.sol#L155 @0xZenus which does not rely on prb math but rather relies on the low level math functions of v3

Reply:
tbh, might be worth packaging the fixed rate to tick conversion logic into a separate internal function that can then be separately tested in UTs with a few sample inputs

Reply:
That logic converts sqrtPrice to fixed rate. Here we need to convert the fixed rate to tick.

Reply:
yeah, that's true, can we not just do the reverse of that logic if that makes sense?